tests/examples/wibox/awidget/prompt/vilike.lua

```diff
@@ -0,0 +1,55 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local awful     = { prompt = require("awful.prompt"),--DOC_HIDE
+                    util   = require("awful.util")}--DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+local terminal  = "xterm"  --DOC_HIDE
+
+    local atextbox = wibox.widget.textbox()
+
+    -- Store a list of verbs characters in a hash
+    local verbs = {
+        -- Spawn in a terminal
+        t = function(adjs, count, cmd) return {terminal, "-e", cmd} end,  --luacheck: no unused args
```

Comment:
It was mentioned somewhere else that the `-e` works differently on terminals.
Have we addressed this somehow already?